### DISCUSSION TITLE
Transferring an issue to a new repository should transfer all assets as well

### DISCUSSION THREAD

**Discussion by abdulwahid-h** [community/community#109805] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I created an issue on a repo A and added an image to its description. I then transferred the issue to repo B. The image URL however still references repo A. When I delete repo A, the image 404s.

I think transferring an issue to a new repo should transfer all its "assets" as well to prevent the 404 on delete.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by fgnievinski** [comment]
similar issue:
https://gitlab.com/gitlab-org/gitlab-foss/-/issues/57825
